README.md

```diff
@@ -472,6 +472,29 @@ Prints string to stdout, and returns string with additional utility methods
 like `.to()`.
 
 
+### mktemp([options,] [templates...])
+
+Available Options:
+
++ `-d`: Create a directory instead of a file.
++ `-u`: Unsafe mode. (Delete the file before retuning). Only use if you know what you're doing.
```

Comment:
Basically, but it makes sure that you have the permissions to create the file.

This would be useful if you had a command which took a path, and put something there, but that path wasn't allowed to exist beforehand. However, it's unsafe because theoretically someone else could put something there first.